xml/art_amd-sev.xml

```diff
@@ -440,6 +440,14 @@ spec:
      PCI passthrough).
     </para>
    </listitem>
+   <listitem>
+    <para>
+     SEV-ES VMs cannot be rebooted from within using <command>reboot</command>,
+     <command>shutdown -r now</command>, etc. A reboot must be done by shutting
+     down the VM and starting it again. This limitation does not apply to SEV
+     VMs, only SEV-ES.
+    </para>
+   </listitem>
   </itemizedlist>
```

Comment:
Do we also have to mention something about Secure Boot for guests? If I understand correctly at least current generation SEV/SEV-ES have some limitation when trying to use them in combination with OVMF firmwares that provide SMM mode support, so are SEV/SEV-ES fundamentally incompatible with Secure Boot? Or what is the right relationship? @joergroedel

Reply:
Secure boot is not enabled by default and it is not a commonly used feature like reboot, save/restore, and migration. I don't see SEV + secure boot not working as a limitation. Also, one of the commits in this PR explicitly lists the two supported firmwares.

Reply:
well with the layman in mind, how is one supposed to know that SEV and secure boot cannot be combined if we don't document it somewhere..? There is explicit list of firmwares, but that assumes that a user makes the two connections on his own (ie which firmwares are required for secure boot, which ones for SEV, and put two and two together..) would it hurt to be explicit?

Reply:
Nod. I'll add another list item stating the two are currently incompatible.

Reply:
Added it in V2.